Getting close now; only one related failure;

```
=== RUN   TestDockerSuite/TestAPIStatsNetworkStats
    docker_api_stats_test.go:145: assertion failed: error is not nil: exec: "ping": executable file not found in $PATH
    --- FAIL: TestDockerSuite/TestAPIStatsNetworkStats (1.75s)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-# syntax=docker/dockerfile:1.2
+# syntax=docker/dockerfile:1.3
 
 ARG CROSS="false"
 ARG SYSTEMD="false"
@@ -21,10 +21,8 @@ ENV GO111MODULE=off
 FROM base AS criu
 ARG DEBIAN_FRONTEND
 ADD --chmod=0644 https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/Release.key /etc/apt/trusted.gpg.d/criu.gpg.asc
-# FIXME: temporarily doing a manual chmod as workaround for https://github.com/moby/buildkit/issues/2114
 RUN --mount=type=cache,sharing=locked,id=moby-criu-aptlib,target=/var/lib/apt \
     --mount=type=cache,sharing=locked,id=moby-criu-aptcache,target=/var/cache/apt \
-        chmod 0644 /etc/apt/trusted.gpg.d/criu.gpg.asc \
         && echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/ /' > /etc/apt/sources.list.d/criu.list \
         && apt-get update \
         && apt-get install -y --no-install-recommends criu \

--- a/Dockerfile
+++ b/Dockerfile
@@ -252,7 +252,6 @@ RUN --mount=type=cache,sharing=locked,id=moby-dev-aptlib,target=/var/lib/apt \
     --mount=type=cache,sharing=locked,id=moby-dev-aptcache,target=/var/cache/apt \
         apt-get update && apt-get install -y --no-install-recommends \
             apparmor \
-            aufs-tools \
             bash-completion \
             bzip2 \
             iptables \

--- a/Dockerfile.simple
+++ b/Dockerfile.simple
@@ -36,7 +36,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		xfsprogs \
 		xz-utils \
 		\
-		aufs-tools \
 		vim-common \
 	&& rm -rf /var/lib/apt/lists/*
 

--- a/integration/network/service_test.go
+++ b/integration/network/service_test.go
@@ -20,6 +20,7 @@ import (
 
 // delInterface removes given network interface
 func delInterface(t *testing.T, ifName string) {
+	t.Helper()
 	icmd.RunCommand("ip", "link", "delete", ifName).Assert(t, icmd.Success)
 	icmd.RunCommand("iptables", "-t", "nat", "--flush").Assert(t, icmd.Success)
 	icmd.RunCommand("iptables", "--flush").Assert(t, icmd.Success)

